src/test/java/fr/insee/pearljam/api/dto/campaign/CampaignSensitivityDtoTest.java

```diff
@@ -0,0 +1,26 @@
+package fr.insee.pearljam.api.dto.campaign;
+
+import fr.insee.pearljam.api.domain.*;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CampaignSensitivityDtoTest {
+
+    @Test
+    @DisplayName("Test of creating a sensitive campaign")
+    void testCreateCampaignSensitivity() {
+        // Given
+        Campaign campaign = new Campaign("id", "label", IdentificationConfiguration.IASCO,
```

Comment:
suggestion: IdentificationConfiguration.IASCO is deprecated => should replace with HOUSEF2F